test/optimus/pr/ValidateOptimus.wdl

```diff
@@ -0,0 +1,62 @@
+
+task ValidateOptimus {
+      File bam
+      File matrix
+      File matrix_summary
+      File picard_metrics
+      Int required_disk = ceil((size(bam, "G") + size(matrix, "G") + size(matrix_summary, "G") + size(picard_metrics, "G")) * 1.1)
+
+      String expected_bam_hash
+      String expected_matrix_hash
+      String expected_matrix_summary_hash
+      String expected_picard_metrics_hash
+      # # these are also inputs; lets leave them alone until flattening is in WDL
+      # Array[Array[File]] tag_gene_exon_log
+      # Array[Array[File]] umi_metrics
+      # Array[Array[File]] duplicate_metrics
+
+  command <<<
+
```

Comment:
you probably want to add
```
set -eo pipefail
```
at the top of your command just so you catch any intermittent failures.